src/views/blocks/tickets/footer.php

```diff
@@ -20,8 +20,16 @@
 $tickets         = $this->get( 'tickets' );
 $currency_symbol = $this->get( 'currency_symbol' );
 $provider        = $this->get( 'provider' );
-$cart_url        = $provider->get_cart_url();
-$checkout_url    = $provider->get_checkout_url();
+if ( method_exists( $provider, "get_cart_url" ) ) {
```

Comment:
:no_entry_sign: String "get_cart_url" does not require double quotes; use single quotes instead

<sub>hash: 20462959028050a27ea029685345d1a9</sub>

Reply:
```suggestion
if ( method_exists( $provider, 'get_cart_url' ) ) {
```